conda-forge.yml

```
@@ -1,3 +1,5 @@
+bot:
+  inspection: hint-all
 build_platform:
   osx_arm64: osx_64
```

Better use cross-compilation, we have a lot of issues with Travis lately.
```suggestion
  osx_arm64: osx_64
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```